spec/workers/weekly_balance_reminder_spec.rb

```diff
@@ -18,15 +18,16 @@
 
   describe '#perform' do
     before do
-      expect(BalanceMailer)
-        .to receive(:debt_email)
-        .with(user, [an_instance_of(Balance::Wrapper)])
-        .and_return(email)
-      expect(email).to receive(:deliver_now)
+      allow(BalanceMailer).to receive(:debt_email).and_return(email)
+      allow(email).to receive(:deliver_now)
     end
 
     it 'sends email to the user' do
       subject.perform
+      expect(BalanceMailer)
+        .to have_received(:debt_email)
+        .with(user, [an_instance_of(Balance::Wrapper)])
```

Comment:
`Balance::Wrapper#user` is a 🐴fusing method name. Because this is the payer, or the user `self` owes money to...